crates/core_arch/src/riscv_shared/mod.rs

```diff
@@ -1,5 +1,29 @@
 //! Shared RISC-V intrinsics
-
+//!
+//! ## Missing floating-point register instructions
+//!
+//! We are deliberately *not* providing instructions that could change the floating-point rounding
+//! mode or exception behavior or read the accrued exceptions flags: `frcsr`, `fscsr`, `fsrm`,
+//! `frflags`, `fsflags`.
+//!
+//! Rust makes no guarantees whatsoever about the contents of the accrued exceptions register: Rust
+/// floating-point operations may or may not result in this register getting updated with exception
```

Comment:
You switched to `///` comment style halfway through.

Reply:
*oops* fixed